examples/lb/members/base-computes.tf

```diff
@@ -0,0 +1,12 @@
+resource "vkcs_compute_instance" "front_worker" {
+  count           = 2
+  name            = "front-worker-${count.index}-tf-example"
+  flavor_id       = data.vkcs_compute_flavor.basic.id
+  security_groups = ["default", "worker_front"]
```

Comment:
do not we have appropriate group in networks main?

Reply:
there are rules in firewall/main but too many of them we do not need those